integrations/dd-sdk-android-okhttp/api/apiSurface

```diff
@@ -23,6 +23,8 @@ open class com.datadog.android.okhttp.DatadogInterceptor : com.datadog.android.o
   override fun onRequestIntercepted(com.datadog.android.api.feature.FeatureSdkCore, okhttp3.Request, io.opentracing.Span?, okhttp3.Response?, Throwable?)
   override fun canSendSpan(): Boolean
   override fun onSdkInstanceReady(com.datadog.android.core.InternalSdkCore)
+data class com.datadog.android.okhttp.DdOtelContext
```

Comment:
you probably can name this something like `TraceContext` (even without `Dd` prefix), because there is nothing from Otel here actually, this class only contains trace ID, span ID and sampling as int, all of these are applicable to the OpenTracing as well.

Reply:
yeah was not sure about the naming